lua/terrortown/lang/zh_hans.lua

```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
---L.label_voice_scaling = "Voice Volume Scaling Mode"
---L.label_voice_scaling_mode_linear = "Linear"
---L.label_voice_scaling_mode_power4 = "Power 4"
---L.label_voice_scaling_mode_log = "Logarithmic"
+L.label_voice_duck_spectator = "降低观察者语音"
+L.label_voice_duck_spectator_amount = "降低观察者语音音量"
+L.label_voice_scaling = "语音音量缩放模式"
+L.label_voice_scaling_mode_linear = "线性"
+L.label_voice_scaling_mode_power4 = "四次方"
+L.label_voice_scaling_mode_log = "对数"
 
 -- 2023-10-25
---L.help_keyhelp = [[
---Key bind helpers are part of a UI element that always shows relevant keybindings to the player, which is especially helpful for new players. There are three different types of key bindings:
---
---Core: These contain the most important bindings found in TTT2. Without them the game is hard to play to its full potential.
---Extra: Similar to core, but you don't always need them. They contain stuff like chat, voice or flashlight. It might be helpful for new players to enable this.
---Equipment: Some equipment items have their own bindings, these are shown in this category.
---
---Disabled categories are still shown when the scoreboard is visible]]
-
---L.label_keyhelp_show_core = "Enable always showing the core bindings"
---L.label_keyhelp_show_extra = "Enable always showing the extra bindings"
---L.label_keyhelp_show_equipment = "Enable always showing the equipment bindings"
-
---L.header_interface_keys = "Key helper settings"
---L.header_interface_wepswitch = "Weapon switch UI settings"
-
---L.label_keyhelper_help = "open gamemode menu"
---L.label_keyhelper_mutespec = "cycle spectator voice mode"
---L.label_keyhelper_shop = "open equipment shop"
---L.label_keyhelper_show_pointer = "free mouse pointer"
---L.label_keyhelper_possess_focus_entity = "possess focused entity"
---L.label_keyhelper_spec_focus_player = "spectate focused player"
---L.label_keyhelper_spec_previous_player = "previous player"
---L.label_keyhelper_spec_next_player = "next player"
---L.label_keyhelper_spec_player = "spectate player"
---L.label_keyhelper_possession_jump = "prop: jump"
---L.label_keyhelper_possession_left = "prop: left"
---L.label_keyhelper_possession_right = "prop: right"
---L.label_keyhelper_possession_forward = "prop: forward"
---L.label_keyhelper_possession_backward = "prop: backward"
---L.label_keyhelper_free_roam = "leave object and roam free"
---L.label_keyhelper_flashlight = "toggle flashlight"
---L.label_keyhelper_quickchat = "open quickchat"
---L.label_keyhelper_voice_global = "global voice chat"
---L.label_keyhelper_voice_team = "team voice chat"
---L.label_keyhelper_chat_global = "global chat"
---L.label_keyhelper_chat_team = "team chat"
---L.label_keyhelper_show_all = "show all"
---L.label_keyhelper_disguiser = "toggle disguiser"
---L.label_keyhelper_save_exit = "save and exit"
---L.label_keyhelper_spec_third_person = "toggle third person view"
+L.help_keyhelp = [[
+按键绑定帮助器是用户界面元素的一部分，它总是向玩家显示相关的按键绑定，这对新玩家特别有帮助。按键绑定有三种不同类型：
```

Comment:
```suggestion
键位助手是一个向玩家显示相关键位的界面元素，对新玩家特别有帮助。键位分为三种类型：
```
Recommend localizing both keybinding and binding to "键位" as "绑定" is somewhat ambiguous and "键位绑定" is too long.